### ISSUE TITLE
Won't run

### ISSUE THREAD

**Issue by gobboo** [Juanan7676/netcoin#1]
Can't get it to run at all is there a clearer guide on what to run where to x file and such :)

**Comment by RedstoneParkour** [comment]
Well...
For a normal node (using `node/node.lua`) you need to have `netcraftAPI.lua` and everything in the `node` dir except `node/node.lua` inside `/usr/lib` (or any other valid path in `package.path`) and have `node/node.lua` inside `/usr/bin` (or any other valid path in `$PATH`)
Then you need the following files in the directory where you will run `node.lua` (it uses the current directory for config files)

- `disks.txt`, consisting of a set of lines that are as follows: `name ',' mountPoint`, where `name` is the name for your disk and `mountPoint` is the point where the disk is mounted, excluding the `/mnt/` part AND without any trailing slash. The first line defines the index disk, the second line defines the wallet disk, and any further lines define block storing disks.
- `contacts.txt`, consisting of an empty serialized table: `{}`
- `lb.txt`, consisting of just `error`
- `nodes.txt`, consisting of an empty serialized table: `{}`

Then:
On the index disk, create `index.txt` consisting of nothing.
On the wallet disk, create `remutxo.txt`, `utxo.txt`, `walletremutxo.txt` and `walletutxo.txt` also consisting of nothing.

Then apply a lot of patches. The program SHOULD run by typing `node`.

TLDR: You need to do a lot.
And yes, I might make a fork with all of the above and a setup command.
Also, for the protocol to work, you need to have gamerule `doDaylightCycle` set to `true`

**Comment by gobboo** [comment]
Yeah seems like a lot, just a lot of code reading it seems to get it setup I'll try this thanks

**Comment by Juanan7676** [comment]
Yep, this is very WIP project and need to document it a bit. Thanks for replying, @kcinnaJlol !

**Comment by Juanan7676** [comment]
I am open to accepting pull requests by the way, I do not have currently a lot of time available to spend on this project, but any improvements are more than welcome!
